### PR TITLE
feat: Add Dope Newscast

### PR DESCRIPTION
**Pull request by msidqi**
## What's new
- New UI for news page with markup based on [this PR](https://github.com/dopedao/dope-monorepo/pull/153)
- Pagination for news
- Mobile responsiveness for news

### CODE REVIEW

**Comment by subimage** [comment]
This looks great so far! I'm not sure if I communicated the specs clearly, so I will try here. I'd like to see the following changes…

- [ ] Your UX should replace the one at `/news` completely
- [ ] Articles should be pulled from Markdown like the current `news` page does
- [ ] When the browser is resized we should modify the layout… Tablet size: 2 columns. Phone size: 1 column.
- [ ] Newspaper should be the full page, not inside Desktop or AppWindow

LMK if you need help with any of these!

File: packages/web/src/features/news/components/DopePostBody/index.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    font-family: Dope !important;
+  }
+`;
+
+type DopePostBodyProps = { posts: PostType[]; hasMore: boolean };
+
+const DopePostBody = ({ posts, hasMore }: DopePostBodyProps) => {
+  const router = useRouter();
+  const isTabletOrMobile = useBreakpointValue({ base: true, md: false });
+  console.log({ isTabletOrMobile });
```
Comment (smakosh) [inline packages/web/src/features/news/components/DopePostBody/index.tsx:31]: 👀

Comment (msidqi) [reply packages/web/src/features/news/components/DopePostBody/index.tsx]: good catch!

**Review by smakosh — APPROVED** [review]
LGTM

**Pull request merged**